lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -172,3 +171,39 @@ def gated_by_required_content(block_key, block_structure, required_content):  #
             return True
 
         return False
+
+    def _generate_special_exam_attempt_context(self, is_practice_exam, is_proctored_enabled,
+                                               is_timed_exam, user_id, course_key, block_key):
+        """
+        Helper method which generates the special exam attempt context.
+        Either calls into proctoring or, if exams ida waffle flag on, then get internally.
+        """
```

Comment:
worth calling out that this is a duplicate of the other helper method and a brief why the duplicate for future readers?